HandleNHNotificationCallActivity/__tests__/handler.test.ts

```diff
@@ -54,11 +60,50 @@ const aDeleteInStalltionMessage: DeleteInstallationMessage = {
   kind: "DeleteInstallation" as any
 };
 
+const aNHConfig = {
+  AZURE_NH_ENDPOINT: envConfig.AZURE_NH_ENDPOINT,
+  AZURE_NH_HUB_NAME: envConfig.AZURE_NH_HUB_NAME
+} as NotificationHubConfig;
+
+const mockTelemetryClient = ({ trackEvent: () => {} } as unknown) as Parameters<
+  typeof getCallNHServiceActivityHandler
+>[0];
+
+/**
+ * Spy on `buildNHService` to return a mocked `NotificationHubService`
+ */
+const buildNHServiceMock = jest
+  .spyOn(notificationhubServicePartition, "buildNHService")
+  .mockImplementation((c: NotificationHubConfig) => {
+    return ({
+      createOrUpdateInstallation: createOrUpdateInstallationSpy,
+      send: notifySpy,
+      deleteInstallation: deleteInstallationSpy
+    } as undefined) as NotificationHubService;
```

Comment:
Prefer unknown for this kind of operations:

```suggestion
    } as unknown) as NotificationHubService;
```

Reply:
Done in [2eadf67](https://github.com/pagopa/io-functions-pushnotifications/pull/4/commits/2eadf6746ab882b6b39c85ff15d64f1a3b9057fb)